.mbedignore

```diff
@@ -1,8 +1,8 @@
-cryptoauthlib/app/
```

Comment:
We shouldn't delete app here. I believe you still need the `tng` code in `app` for certificate decompression. Please remove the commit "Do not ignore 'app' directory since it provides the interface for constructing X509 certificates and more" (96bbd884d77f) from this PR.